app/buildpacks/node_buildpack.rb

```diff
@@ -1,8 +1,8 @@
 class NodeBuildpack
   include Buildpack
 
-  LATEST    = '8'.freeze # 8 is the LTS version
-  SUPPORTED = %w(5 6 8 10).freeze
+  LATEST    = '12'.freeze # 8 is the LTS version
```

Comment:
Please update the comment on this line as well. Or remove it entirely.

Reply:
good catch, thank you